docs/3.0rc/concepts/artifacts.mdx

```diff
@@ -102,6 +97,53 @@ In the above example, the `create_link_artifact` method is used within a flow to
 The `link` parameter is used to specify the external resource to be linked to, and `link_text` is used to specify the text to be displayed for the link.
 An optional `description` could also be added for context.
 
+### Creating progress artifacts
+
+Progress artifacts are useful for tracking the progress of long-running tasks. They render dynamically on the flow run timeline in the Prefect UI, providing a visually rich indicator of a task run's progress.
+
+To create a progress artifact, use the `create_progress_artifact()` function. To update the progress of a progress artifact, use the `update_progress_artifact()` function.
+
+![Progress artifact example](/3.0rc/img/ui/progress-artifact-example.png)
+
+```python
+from time import sleep
+
+from prefect import flow, task
+from prefect.artifacts import (
+    create_progress_artifact,
+    update_progress_artifact,
+)
+
+
+def fetch_batch(i: int):
+    # Simulate fetching a batch of data
+    sleep(2)
+
+
+@task
+def fetch_in_batches():
+    progress_artifact_id = create_progress_artifact(
+        progress=0.0,
+        description="Indicates the progress of fetching data in batches.",
+    )
+    for i in range(1, 11):
+        fetch_batch(i)
+        update_progress_artifact(artifact_id=progress_artifact_id, progress=i * 10)
+
+
+@flow
+def etl():
+    fetch_in_batches()
+
+
+if __name__ == "__main__":
+    etl()
+
+```
+
+Note that progress artifacts are updated using a separate `update_progress_artifact()` function. Prefect updates a progress artifact in place, rather than versioning it.
```

Comment:
```suggestion
Progress artifacts are updated using a separate `update_progress_artifact()` function. Prefect updates a progress artifact in place, rather than versioning it.
```